packages/components/pie-lottie-player/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import {
+    LitElement, html, isServer, unsafeCSS,
+} from 'lit';
+import { property, query } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { LottiePlayer, AnimationItem } from 'lottie-web';
+
+import { defineCustomElement } from '@justeattakeaway/pie-webc-core';
+import { LottiePlayerProps, defaultProps } from './defs';
+
+// Valid values available to consumers
+export * from './defs';
+
+const componentSelector = 'pie-lottie-player';
+
+/**
+ * @tagname pie-lottie-player
+ */
+export class PieLottiePlayer extends LitElement implements LottiePlayerProps {
+    @query('div')
```

Comment:
This evidently works well, but it feels perhaps a little more solid and reliable to hook onto an id or an attribute rather than `div`. Not the most important thing but thought I'd mention it.

Reply:
Sure, I think other components are a little more specific like `input[type="checkbox"]` or `textarea` rather than just `div`. We can update later if it starts causing problems.